web-development/TodoList/app.js

```diff
@@ -0,0 +1,166 @@
+// link - https://calm-refuge-93371.herokuapp.com/
+
+const express = require("express");
+const bodyParser = require("body-parser");
+const mongoose = require("mongoose");
+const _ = require("lodash");
+const date = require(__dirname + "/date.js");
+
+const app = express();
+
+app.set('view engine', 'ejs');
+
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(express.static("public"));
+
+mongoose.connect("mongodb://localhost:27017/todolistDB"); // connecting database
```

Comment:
How can someone setup this in their local? I would suggest to remove the server files from this repo. We would create a new repo for nodejs within today. Add these files there